zenoh/src/lib.rs

```diff
@@ -227,10 +227,10 @@ where
 {
     OpenBuilder {
         config,
-        publications_destination: Locality::Any,
-        subscribers_origin: Locality::Any,
-        queries_destination: Locality::Any,
-        queryables_origin: Locality::Any,
+        publications_destination: Locality::default(),
+        subscribers_origin: Locality::default(),
+        queries_destination: Locality::default(),
```

Comment:
`Locality` should be only on subscriber and queryable side for the time being at least.